internal/cmd/cmd_test.go

```diff
@@ -180,8 +192,9 @@ XlW7KilKI5YkcszGoPB4RePiHsH+7trf7l8IQq5r5kRq7SKsZ41BI6s1E1PQVW93
 
 		assert.Equal(t, len(kubeconfig.Clusters), 2)
 		assert.Equal(t, len(kubeconfig.Contexts), 2)
-		assert.Equal(t, len(kubeconfig.AuthInfos), 2)
+		assert.Equal(t, len(kubeconfig.AuthInfos), 1)
```

Comment:
We should update the example config to include an `AuthInfos` that we are not modifying, to show that we preserve other entries. 

I think that's why all the other cases here have 2 entries.

Reply:
oh good point. I'll make another issue for that.